tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,222 @@ keywords:
+            int j = i - 1;
+            
+            // Comparing nums with all the elements left to it
+            while (j >= 0 and arr[j] > nums) { 
+                // Shifting the greater element to the right
+                arr[j + 1] = arr[j]; 
+                j--;
+            }
+            // Placing selected element at correct position
+            arr[j + 1] = nums; 
+        }
+    }
+
+    int maxProduct(vector<int>& nums) {
+        insertionSort(nums,nums.size());
+        int n = nums.size();
+        return (nums[n - 1] - 1)*(nums[n - 2] - 1);
+    }
+};
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+<SolutionAuthor name="@Shivashish-rwt"/>
+
+```java
+class Solution {
+    void insertion_sort(int arr[],int size_of_array) {
+		for (int i = 1; i < size_of_array; ++i) {
+            // Picking element from array
+			int nums = arr[i]; 
+			int j = i - 1;
+
+            // Comparing nums with all the elements left to it
+			while (j >= 0 && arr[j] > nums) { 
+                // Shifting the greater element to the right
+				arr[j + 1] = arr[j]; 
+				j = j - 1;
+			}
+            // Placing selected element at correct position
+			arr[j + 1] = nums; 
+		}
+	}
+    public int maxProduct(int[] nums) {
+        insertion_sort(nums,nums.length);
+        int n = nums.length;
+        return (nums[n - 1] - 1)*(nums[n - 2] - 1);      
+    }
+}
+```
+
+</TabItem>
+
+<TabItem value="py" label="Python">
+<SolutionAuthor name="@Shivashish-rwt"/>
+
+```py
+class Solution:
+    def insertionSort(arr,size_of_array):
+        
+        for i in range(1, size_of_array):
+            # Picking element from array
+            nums = arr[i]  
+            j = i - 1
+              
+            # Comparing nums with all the elements left to it  
+            while j >= 0 and arr[j] > nums: 
+                # Shifting the greater element to the right
+                arr[j + 1] = arr[j] 
+                j = j - 1
+
+            # Placing selected element at correct position
+            arr[j + 1] = nums 
+        
+
+    def maxProduct(self, nums: List[int]) -> int:
+        Solution.insertionSort(nums,len(nums))
+        return (nums[-1] - 1) * (nums[-2] - 1)
+```
+
+</TabItem>
+
+</Tabs>
+
+## Time Complexity
+
+Apart from sorting the the given array we are not doing anything in the solution. The insertion sort would take $O(n^2)$ to sort the array.
+
+So, the Time complexity is = $O(n^2)$
+
+## Space Complexity
+
+We are not using any extra space apart from the array we have to sort. So, the space complexity is = $O(1)$
+
+export const suggestedProblems = [
+  {
+    "problemName": "1365 - How Many Numbers Are Smaller Than the Current Number",
+    "difficulty": "Easy",
+    "leetCodeLink": "https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/",
+    "solutionLink": "([Click for solution](/solutions/1300-1399/1365-how-many-numbers-are-smaller-than-the-current-number-easy.md)"
```

Comment:
I'm pretty sure you haven't tested and this is 100% not working.

please look at how other pages for reference and TEST it on your local first before asking for review.

Reply:
I have tested it locally and the links are working fine. I don't know why is it not working on ur setup.

Reply:
I'm talking about `solutionLink` on L210.  The format is already wrong.

Reply:
I wanna ask is this is correct format?
../../../solutions/1300-1399/how-many-numbers-are-smaller-than-the-current-number-easy.md

Reply:
I have tested it and link is working perfectly fine.